library/Stratosphere/Types.hs

```diff
@@ -1,233 +0,0 @@
-{-# LANGUAGE DeriveAnyClass #-}
-
--- | Module for hand-written types that are used in generated modules.
-
-module Stratosphere.Types
-  ( ApiBackendType(..)
-  , AttributeType(..)
-  , AuthorizationType(..)
-  , AuthorizerType(..)
-  , CannedACL(..)
-  , EnabledState(..)
-  , HttpMethod(..)
-  , KeyType(..)
-  , KinesisFirehoseElasticsearchS3BackupMode(..)
-  , KinesisFirehoseNoEncryptionConfig(..)
-  , KinesisFirehoseS3CompressionFormat(..)
-  , LoggingLevel(..)
-  , PassthroughBehavior(..)
-  , Period(..)
```

Comment:
Many of these types are service specific, but do not have service specific names. Refactoring this would have broken interface anyway. So I decided to do away with it (see PR/commit message) for the moment.